session_srtp.go

```diff
@@ -53,12 +55,6 @@ func NewSessionSRTP(conn net.Conn, config *Config) (*SessionSRTP, error) {
 	return s, nil
 }
 
-// Start initializes any crypto context and allows reading/writing to begin
```

Comment:
This is actually started by initializing the session, we shouldn't expose this API anymore since it could cause double starts.